### PR TITLE
Update integration sample - Google analytics

### LINKED ISSUE #146: Update integration sample - Google analytics
### Brief bug description

After the [rebranding from Kentico Cloud to Kentico Kontent](https://kontent.ai/blog/moving-to-caas-with-kentico-kontent) there are still some samples that use old naming in branches named `samples/*`.

Until the end of October endpoint on domain `kenticocloud.com` stop working and there will be redirect performed:
* https://docs.kontent.ai/reference/api-changelog#a-api-urls-changed

#### Transfer changes made in https://github.com/Kentico/kontent-sample-app-react/pull/140 to the [**`samples/ga`**](https://github.com/Kentico/kontent-sample-app-react/tree/samples/ga) branch

* Update dependency from `kentico-cloud-delivery` to `@kentico/kontent-delivery`
    * adjust the breaking changes
* Update dependency `@chevtek/react-spinners` to `@simply007org/react-spinners`
   * no breaking changes
* Rename Kentico Cloud to Kentico Kontent
* adjust `travis.yml` to use the correct domain for deployment

### PR DESCRIPTION
**Pull request by uchemukolo**
### Motivation

Which issue does this fix? Fixes #146

If no issue exists, what is the fix or new feature? Were there any reasons to fix/implement things that are not obvious?

### Checklist

- [ ] Code follows coding conventions held in this repo
- [ ] Automated tests have been added
- [ ] Tests are passing
- [ ] Docs have been updated (if applicable)
- [ ] Temporary settings (e.g. variables used during development and testing) have been reverted to defaults

### How to test

If manual testing is required, what are the steps?

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -9,6 +9,6 @@ script:
 deploy:
   provider: surge
   project: ./build/
-  domain: https://kentico-cloud-sample-app-react-ga.surge.sh
+  domain: https://kontent-sample-app-react.surge.sh
```
Comment (Simply007) [inline .travis.yml]: Please use -ga suffix:

```suggestion
  domain: https://kontent-sample-app-react-ga.surge.sh
```

**Comment by uchemukolo** [comment]
@Simply007 Please where can i fine `Removal of /examples/* and /docs/*` in the code

**Comment by uchemukolo** [comment]
@Simply007 all changes have been implemented. since the `home` variable is not available in this file, I used the `metaData` variable instead. hope this is okay,

File: src/Pages/Home.js [hunk 2]
```diff
@@ -67,7 +67,10 @@ class Home extends Component {
           twitterImage={metaData.metadataTwitterImage}
         />
         <Banner />
-        <LatestArticles language={this.props.language} />
+        <LatestArticles
+          articles={metaData.articles.value}
```
Comment (Simply007) [inline src/Pages/Home.js]: please revert this loading
```suggestion
```

Comment (uchemukolo) [reply src/Pages/Home.js]: @Simply007 I have reverted as suggested.

**Pull request merged**